contracts/src/storage/AssetsStorage.sol

```diff
@@ -15,6 +15,8 @@ library AssetsStorage {
         uint128 assetHubReserveTransferFee;
         // Extra fee for registering a token, to discourage spamming (Ether)
         uint256 registerTokenFee;
+        /// The maximum fee that can be sent to a destination chain during a reserve transfer (DOT)
+        uint128 destinationMaxTransferFee;
```

Comment:
So is this set per agent in storage? In theory every agent could have a different limit?

Reply:
There is only one `struct Layout` instance used for all agents.